_transcripts/013-rust-1.41.0.md

```diff
@@ -0,0 +1,800 @@
+---
+title: "What's New in Rust 1.41"
+file:
+https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e013-rust-1.41.0.mp3
```

Comment:
Oh, just barely caught this: there can't be a line-break here!

Reply:
Fixed.

Reply:
In the future, I'll try to look at the Jekyll log (I have it running in `--watch` mode in another window) before sending the PR.